tests/features/api/reverse_simple.feature

```diff
@@ -50,3 +50,4 @@ Feature: Simple Reverse Tests
         Then the result is valid json
         When looking up jsonv2 coordinates 36.791966,127.171726
         Then the result is valid json
+
```

Comment:
Nitpicking here but if you could get rid of this empty line change, that would be nice.